### PR TITLE
Merge gsd upp

### LINKED ISSUE #158: Merge gsd UPP
Incorporate the changes of rapv5/hrrrv4 from gsd upp repository at VLAB.

### PR DESCRIPTION
**Pull request by WenMeng-NOAA**
Incorporate the changes of rapv5/hrrr4 from GSD UPP repository on VLAB:
1) new or modified GSD algorithms in source code;
2) new fields in params_grib2_tbl_new;
3) post_avblflds_raphrrr.xml, and control files for hrrr and rap;
4) clean some legacy control files for hrrr and rap.

### CODE REVIEW

**Comment by WenMeng-NOAA** [comment]
@hu5970 and @jaymes-kenyon Please review the code and let me know anything is missing or incorrect. Thanks!

**Review by jaymes-kenyon — COMMENTED** [review]
Looks good to me, Wen.  It's a very large merge, so I'd like Ming to review this, also.  Thanks for your work here!

File: sorc/ncep_post.fd/CLDRAD.f [hunk 1]
```diff
@@ -1060,6 +1062,29 @@ SUBROUTINE CLDRAD
       ENDIF  
 !
 !***  BLOCK 2.  2-D CLOUD FIELDS.
+
+! GSD maximum cloud fraction in (PBL + 1 km) (J. Kenyon, 8 Aug 2019)
```
Comment (WenMeng-NOAA) [inline sorc/ncep_post.fd/CLDRAD.f:1066]: @jaymes-kenyon and @hu5970 , I have been working on setting up new baselines for hrrr and rap. It turns out cloud base height and pressure (ID 711 and 798) are not reproducible with different tasks in both hrrr and rap tests. Do you think anything is missing or incorrect in my syncing version?

Comment (jaymes-kenyon) [reply sorc/ncep_post.fd/CLDRAD.f:1066]: Hi Wen.  This is surprising, and I'm not yet sure on the reason.  One possibility:  in GSD's HRRR runs on Jet, we experienced occasional UPP crashes, and we found that lowering the fill value for the CLDZ(i ,j) array fixed this.  For example, after CLDZ is passed to GRID1, we added:  

GRID1(I,J) = max(min(GRID1(I,J), 20000.0),0.0)

This change was only made in GSD's UPP master branch on VLab, so it's not in your synched code.  

Another possibility is that the cloud base height/pressure algorithm has a bug.  It uses a horizontal search of nearby grid points (the "ceil_neighbor") array, which might not be parallelizing correctly.  I'm ready to help if we believe it's a bug.

**Review by hu5970 — APPROVED** [review]

**Comment by WenMeng-NOAA** [comment]
The UPP regression tests for GFS, GEFS, NMMB, FV3SAR and HAFS were conducted on Dell and Hera. No changed results were found. New baselines for HRRR and RAP were created. It was found that cloud base height and pressure were not reproducible with  changing MPI tasks. But these two fields will not have an impact on other application processing. The bug fix would be added in the future. This branch post_gsd is ready for merging.

**Comment by WenMeng-NOAA** [comment]
Re-create the regression baselines for NMMB, HRRR, RAP.

**Pull request merged**